shell/browser/javascript_environment.h

```diff
@@ -46,9 +46,8 @@ class JavascriptEnvironment {
   node::MultiIsolatePlatform* platform_;
 
   v8::Isolate* isolate_;
-  gin::IsolateHolder isolate_holder_;
+  std::unique_ptr<gin::IsolateHolder> isolate_holder_;
```

Comment:
What is the benefit of changing this from a `gin::IsolateHolder` to a `std::unique_ptr<gin::IsolateHolder>`?